watch-parties/src/main/webapp/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Watch Parties</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <div id="queue">
+      <form>
+        <input type="text" id="videoLink">
+        <input type="button" id="queueSubmit" value="Add Link">
+      </form>
+      <table>
```

Comment:
I suggest using `div` and `span` for just about everything instead of `table`. See this example for an idea of a cleaner way to get this layout: https://jsfiddle.net/nLrwf1e0/2/